uaaextras/templates/email/body.html

```diff
@@ -17,18 +17,18 @@
         <p>
         <ol>
             <li>
-            <b>Accept the invitation</b> - <a href="{{verification_url}}">Accept your invite</a> to continue the registration process. (The invitation will expire after 24 hours.) You can also copy the URL below and paste it into your browser's address bar:
```

Comment:
Let's change the `<b>` tags to `<strong>` throughout, for consistency and semantic accuracy.

Reply:
Updated.